test/bdd/vdr/orb/fixtures/orb/docker-compose.yml

```diff
@@ -35,6 +36,7 @@ services:
       - KMSSECRETS_DATABASE_PREFIX=orb1kms\
       - ORB_AUTH_TOKENS_DEF=/services/orb/keys,/services/orb/outbox|admin&read|admin,/services/orb/inbox|admin&read&admin|admin,/services/orb/.*|read&admin,/transactions|read&admin
```

Comment:
Did you want to add side-tree permissions:
...,/sidetree/.*/identifiers|read&admin,/sidetree/.*/operations|read&admin|admin

https://github.com/trustbloc/orb/blob/759547cbf041f6bf3f19e99fafadc8a6490ed8cb/test/bdd/fixtures/docker-compose.yml#L61

Reply:
No need to test it in vdr